src/sentry/api/endpoints/monitor_checkin_details.py

```diff
@@ -100,6 +117,27 @@ def get(self, request: Request, project, monitor, checkin) -> Response:
 
         return self.respond(serialize(checkin, request.user))
 
+    @extend_schema(
+        operation_id="Update a check-in",
+        parameters=[
+            MONITOR_PARAMS.MONITOR_ID,
```

Comment:
Should we include the organization slug in the docs? We'll eventually need to remove the URLs that don't have an organization slug.

Reply:
Good catch - not sure what the strategy is here since this endpiont is registered at two paths.. I guess since this is just docs I can jus tignore the one without the org slug for this call